infrastructure/prod-deploy/update-content.sh

```diff
@@ -16,5 +16,6 @@ source ./codewof/load-prod-envs.sh
 ./dev migrate
 docker-compose exec django /docker_venv/bin/python3 ./manage.py load_user_types
 ./dev load_questions
+./dev load_style_errors
```

Comment:
Are we not adding this to dev-deploy as well?

Reply:
This is included in the `sampledata` command.